src/lib/journey/callbacks/string-attribute/string-attribute-input.svelte

```diff
@@ -35,10 +36,21 @@
      *********************************************************************** */
     callback.setInputValue((event.target as HTMLInputElement).value);
   }
+
+  $: {
+    isRequired = isInputRequired(callback);
+    outputName = callback.getOutputByName('name', '');
+    policies = callback.getPolicies();
+    type = getInputTypeFromPolicies(policies);
+    previousValue = callback?.getInputValue() as string;
+    textInputLabel = callback.getPrompt();
+    validationFailure = getAttributeValidationFailureText(callback);
+  }
```

Comment:
It's worth noting that with Svelte, if you have a local variable that is assigned a value through the calling of a function, you have to wrap those assignments in a reactive block. This is because Svelte won't rerun all function within a component, so you need to explicitly tell Svelte to rerun the ones you want.

Ordinary, direct, variable assignment is observed by default, so it's just assignment through functions to be mindful of.